requirements.md

```diff
@@ -0,0 +1,9 @@
+# FishSense Lite System Requirements
+- [ ] Maximum effective distance: 3 m
+- [ ] Minimum effective distance: 0.5 m? (driven)
+- [ ] Spatial resolution at max distance: < 1 mm
+- [ ] Depth resolution at max distance: ??? (driven)
+- [ ] Maximum length error of 5 cm object at max distance: < 5%
```

Comment:
I'm not sure if this is the requirement for this device.

Reply:
What is the appropriate requirement for FSL?